README.md

```diff
@@ -1,23 +1,21 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Portfolio
 
-> One paragraph statement about the project.
+> This is the first project of my portfolie. It has no functionally as yet.
 
-![screenshot](./app_screenshot.png)
+![screenshot](../image/Illustration.png)
```

Comment:
- [ ] Hi, nice work on the required changes from the previous reviewer. However, the path to this image is broken. Consider using the right part for the image and make sure the image name is correct 😄.

Your screenshot:
<img width="191" alt="chrome_wr8ZfIYVxz" src="https://user-images.githubusercontent.com/81468675/144296733-fc2f8ad7-eafc-453d-ae4a-cc6ede469381.png">